packages/flutter/lib/src/widgets/scrollable_helpers.dart

```diff
@@ -160,8 +160,8 @@ class EdgeDraggingAutoScroller {
   EdgeDraggingAutoScroller(
     this.scrollable, {
     this.onScrollViewScrolled,
-    this.velocityScalar = _kDefaultAutoScrollVelocityScalar,
-  });
+    double? velocityScalar,
+  }): velocityScalar = velocityScalar ?? _kDefaultAutoScrollVelocityScalar;
```

Comment:
Can you document that the scalar will be 7 if not set or null

Reply:
Done